README.rst

```diff
@@ -32,6 +32,16 @@ Installation
 See doc/install.txt in the source code archive.
 Python 2.7.2 or later is needed.
 
+
+Install using Docker
+---------------------
+You can try linkchecker in an isolated enviroment without changing any local files or have to install all the dependencies via a Docker Container:
+
+``docker pull ellerbrock/linkchecker``
+
+https://hub.docker.com/r/ellerbrock/link-checker/
```

Comment:
hi long story short: you can run link-checker without the pain to install all the dependencies on the local machine in an isolated enviroment. in case there is an "offical" docker hub account i can make a pull request with the dockerfile itself if you like. this one gets maintained by me and an example how to use it for ci stuff you can see here:
https://github.com/ellerbrock/travis-link-checker

cheers